apps/web/src/data/ecosystem.json

```diff
@@ -1198,8 +1198,8 @@
   },
   {
     "name": "Alien Base",
-    "url": "https://app.alienbase.xyz/",
-    "description": "Alien Base is the Base-native decentralized exchange ecosystem. Use it to trade blue chip assets and derivatives, or check out Area 51 to play with experimental tokens, predictions and lotteries",
+    "url": "https://alienbase.xyz/",
+    "description": "Alien Base is building the Base trading hub for efficiently trading tokens and memecoins onchain. Offering aggregated liquidity, limit orders, yield farming and more, Alien technology helps you use Base to the fullest.",
```

Comment:
Please update:

`building the Base trading hub`

To:

`building a Base trading hub`

---

Additionally, please revise to drop the superlative, `to the fullest`